lessons/r/dplyrmagrittr/lesson.Rmd

```diff
@@ -0,0 +1,351 @@
+---
+layout: page
+title: Data wrangling with dplyr and magrittr
+visible: true
```

Comment:
Almost there! Just need to change `true` to `false` here in the Rmd file. (not the md file though).